pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -1,41 +1,39 @@
 package ovirt
 
 import (
-	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 
 	"github.com/openshift/installer/pkg/types/ovirt"
+	ovirtsdk "github.com/ovirt/go-ovirt"
 )
 
 // Platform collects ovirt-specific configuration.
 func Platform() (*ovirt.Platform, error) {
 	p := ovirt.Platform{}
 
+	var c *ovirtsdk.Connection
 	ovirtConfig, err := NewConfig()
+	if err == nil {
```

Comment:
Why just ask once? I mean if we are giving the user the ability to ask fix credentials...
Something like 

```
for i := 0; i < MAX_RETRY; i++ {
		if err != nil {
			ovirtConfig, err = engineSetup()
			if err != nil {
				return nil, err
			}
		}
		c, err = ovirtConfig.getValidatedConnection()
		if err != nil {
			logrus.Warnf("failed to validate oVirt configuration (%v)", err)
		} else {
			break
		}
	}
```

Reply:
@Gal-Zaidman how's this? (See latest push.)